diff -Naur1 libevdev/common.mk libevdev-cflags/common.mk
--- libevdev/common.mk	2013-10-16 05:49:34.680004835 +0000
+++ libevdev-cflags/common.mk	2013-10-16 05:50:38.145004865 +0000
@@ -278,5 +278,4 @@
 # The same goes for CFLAGS.
-COMMON_CFLAGS-gcc := -fstack-protector-strong -fvisibility=internal -ggdb3 \
-  -Wa,--noexecstack
-COMMON_CFLAGS-clang := -fstack-protector-all -fvisibility=hidden -ggdb
+COMMON_CFLAGS-gcc := -fvisibility=internal -ggdb3 -Wa,--noexecstack
+COMMON_CFLAGS-clang := -fvisibility=hidden -ggdb
 COMMON_CFLAGS := -Wall -Werror -fno-strict-aliasing -O1 -Wformat=2
